app/gui/qt/SonicPi.pro

```diff
@@ -117,7 +117,7 @@ RESOURCES += \
 RC_FILE = SonicPi.rc
 
 ICON = images/app.icns
-LIBS         += -lqscintilla2 -lmp3lame
+LIBS         += -lqscintilla2 `pkg-config --libs sndfile`
```

Comment:
Do we need to add `pkg-config` to the build dependencies for this to work every time?